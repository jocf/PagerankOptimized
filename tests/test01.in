2
0.99
4
A
B
C
D
5
D A
D B
D C
B A
B C
