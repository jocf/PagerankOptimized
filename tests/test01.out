A 0.0050
B 0.0033
C 0.0050
D 0.0025
